resource/layout/settingssubstreaming.layout

```diff
@@ -90,18 +106,37 @@
 			spacing=10
 		}
 
-		place { control="Pair" start=RemoteClientList dir=down margin-top=10}
-		place { control="UnPair,SetPIN" start=Pair dir=right spacing=20 margin-left=20 }
+		place { control=Pair start=RemoteClientList dir=down margin-top=10}
+		place { control="UnPair,SetPIN" start=Pair dir=right spacing=20 margin-left=20 }	
+
+		place
+		{
+			start=Pair
+			control=P2PScopeLabel
+			dir=down
+			width=max
+			margin-top=20
+		}
+		place
+		{
+			start=P2PScopeLabel
+			control=P2PScope
+			dir=down
+			width=max
```

Comment:
Default skin makes this width=max, but all the other combo boxes in settings have a set width. Probably set this to ~360 or so.

Reply:
Set to 360.